docs/source/notes/cuda.rst

```diff
@@ -61,3 +63,19 @@ call. This can be used to overlap data transfers with computation.
 
 You can make the :class:`~torch.utils.data.DataLoader` return batches placed in
 pinned memory by passing ``pinned=True`` to its constructor.
+
+.. _cuda-nn-dataparallel-instead:
+
+Use nn.DataParallel instead of multiprocessing
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Most use cases involving batched input and multiple GPUs should default to using
+:class:`~torch.nn.DataParallel` to utilize more than one GPU. Even with the GIL,
+a single python process can saturate multiple GPUs, though at very large numbers
+of GPUs (8+) utilization might drop. Test your use case before investing the
```

Comment:
Sam is working on the 8+ GPU case, in the next major release, utilization wouldn't drop in this case either, just fyi on what's coming next.